src/main/java/nextstep/subway/line/domain/Color.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.subway.line.domain;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import nextstep.subway.common.constant.ErrorCode;
+import nextstep.subway.utils.StringUtils;
+
+@Embeddable
+public class Color {
+
+    @Column(nullable = false)
+    private String color;
```

Comment:
```suggestion
    private final String color;
```
[VO는 보통 불변으로 만들어주어서](https://tecoble.techcourse.co.kr/post/2020-06-11-value-object/) 이렇게 변경해주셔도 좋을 것 같습니다 ㅎㅎ

Reply:
그렇네요! VO 내부의 값을 바꾸지 않고 새로 등록할테니, final이 좋을 것 같습니다 :) 
해당 문서 읽어보고 2단계에 적용해보겠습니다 ㅎㅎ